core/trezor-crypto/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ target_include_directories(TrezorCrypto PUBLIC chacha20poly1305)
 target_include_directories(TrezorCrypto PUBLIC ed25519-donna)
 
 target_compile_options(TrezorCrypto PRIVATE "-std=c99")
+add_definitions(-Wno-cast-qual -Wno-incompatible-pointer-types -Wno-missing-prototypes)
```

Comment:
does this apply to trezor crypto or external crypto?

Reply:
It applies to the former. Note that this is the cmake file for trezor-crypto.